src/render/boolean.ts

```diff
@@ -1,6 +1,15 @@
-export function isTruthy (val: any): boolean {
-  return !isFalsy(val)
+import { Context } from '../context/context'
+
+export function isTruthy (val: any, ctx: Context): boolean {
+  if ((val === null) && ctx && ctx.opts && ctx.opts.jsTruthy ) return false;
+  return !isFalsy(val, ctx)
 }
-export function isFalsy (val: any): boolean {
-  return val === false || undefined === val || val === null
+
+export function isFalsy (val: any, ctx: Context): boolean {
+  if(ctx && ctx.opts && ctx.opts.jsTruthy) {
+    return val == false
```

Comment:
I believe it should be `return !val`. An example is `[] == false` is true, which is not correct.